src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+        <p className="Person__age">
+            {`I am ${person.age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {!person.isMarried && 'I am not married'}
+        {person.isMarried
+            && `${person.partnerName} is my ${person.sex === 'm'
+              ? 'wife' : 'husband'}`}
```

Comment:
Let's create a separate variable for partner:
`const partner = person.sex === 'm' ? 'wife' : 'husband'`

And use it here



{person.isMarried
  ? `${person.partnerName} is my ${partner}
  : 'I am not married'
}